notebooks/arb_dao_grant_distribution/STIP_Updates/12-1-2024/12-1-2024.md

```diff
@@ -0,0 +1,86 @@
+As per the STIP rules all protocols must post bi-weekly forum updates. The second updates will be due by this Friday, January 12th at 12:00PM EST
+
+# Balancer Bi-Weekly Update 12-1-2024
+![arb-forum-update-4.png](arb-forum-update-4.png)
+## Recap of the Previous Two Weeks
+
+
+**ARB Received Last Disbursement:** 342,858 (KYC delays resulted in collecting 2 biweekly payments at the start of the program, details in change section below)
```

Comment:
I think we were paying out 205k a round. The tx here shows us getting 171,429 The tx link below has us claiming 171,429 from hedgey then paying out 205,714, thereby lowering the standard balance in the msig. For the next rounds we'll keep getting 171,429 but pay out less than that, so the standing balance will go up, then after another round or 2 we stop getting more from hedgey, and we burn down standing balance.  I think you need to rethink this paragraph, numbers are off.